templates/partial/_navigation.html

```diff
@@ -28,33 +28,26 @@
           <a class="p-navigation__link" href="#learn-link-menu" aria-controls="learn-link-menu">Learn</a>
           <ul class="p-navigation__dropdown" id="learn-link-menu" aria-hidden="true">
             <li>
-              <div class="p-navigation__dropdown-item is-title">
-                <div class="p-muted-heading u-no-margin--bottom">
-                  Documentation
-                </div>
-              </div>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://juju.is/docs/olm">OLM docs: Manage charms</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://juju.is/docs/sdk">SDK docs: Build charms</a>
-            </li>
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://juju.is/docs/dev">Dev docs: Contribute to the OLM</a>
-            </li>
-            {#<li>
-              <a class="p-navigation__dropdown-item" href="https://charmhub.io/interfaces">Interfaces</a>
-            </li>#}
-            <li>
-              <a class="p-navigation__dropdown-item" href="https://juju.is/tutorials">Tutorials</a>
-            </li>
-            <li>
-              <div class="p-navigation__dropdown-item is-title">
-                <div class="p-muted-heading u-no-margin--bottom">
-                  Resources
-                </div>
-              </div>
+              <a href="https://juju.is/docs" class="p-navigation__dropdown-item">
+                Documentation
+              </a>
+              <ul class="p-navigation__sub-list">
```

Comment:
It doesn't look like this class is doing anything so we can remove it

Reply:
Do you mean "p-navigation__dropdown-item"? After removing it, the format is changed!

Reply:
Sorry, highlighted the wrong line. I meant `p-navigation__sub-list`

Reply:
No worries! `p-navigation__sub-list ` was used for sub-page indentation for the following docs pages.

Reply:
I removed that class! Will fix the hovering effect :)